### ISSUE TITLE
Error when uninstalling multiple flatpak and one of them is not installed

### ISSUE THREAD

**Issue by lachhebo** [flatpak/flatpak#3513]
## Linux distribution and version
Fedora 31 
## Flatpak version
Flatpak 1.4.4
## Description of the problem

When uninstalling multiple packages and one of them is not installed in the system; flatpak return an error as it should be a warning; futhermore, it does try to uninstall the other packages of the list.

## Steps to reproduce

```shell
flatpak uninstall org.videolan.VLC org.keepassxc.KeePassXC -y  --noninteractive
>>> error: org.videolan.VLC/*unspecified*/*unspecified* not installed
```

**Comment by gperconti** [comment]
Solved in PR #3545 https://github.com/flatpak/flatpak/pull/3545

**Comment by lachhebo** [comment]
Thanks :+1:
